### PR TITLE
Refine introduction to compliant authenticators

### LINKED ISSUE #1096: Clarify description of compliant authenticators in Introduction
The paragraph beginning "Broadly, compliant authenticators" is unclear. It should be reworded to state that implementation is possible in software executing on (a) a general purpose computing device, (b) an on-device Trusted Platform Module (TPM) or a Secure Element (SE), or (c) off-device.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #1096

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/webauthn/pull/1109.html" title="Last updated on Nov 28, 2018, 5:30 PM GMT (f63ef89)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1109/742c773...selfissued:f63ef89.html" title="Last updated on Nov 28, 2018, 5:30 PM GMT (f63ef89)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -264,12 +264,14 @@ infrastructure which allows those credentials to be used with {{CredentialsConta
...
-a Trusted Platform Module (TPM), or a Secure Element (SE) integrated into the [=client device=] in conjunction with some means
-for [=user verification=], along with appropriate driver software to mediate access to these components' functionality. Other
-authenticators might operate autonomously from the [=client device=] running the user agent, and be accessed over a transport such
-as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communications (NFC).
+[=Web Authentication API=].
+Implementing compliant authenticators is possible in software executing
+(a) on a general-purpose computing device,
+(b) on an on-device Secure Execution Environment, Trusted Platform Module (TPM), or a Secure Element (SE), or
+(c) off device.
+Authenticators being implemented on device are called Platform Authenticators.
```
Comment (emlun) [inline index.bs]: s/Platform Authenticators/[=platform authenticators=]/ ?

File: index.bs [hunk 2]
```diff
@@ -264,12 +264,14 @@ infrastructure which allows those credentials to be used with {{CredentialsConta
...
+Authenticators being implemented off device (Roaming Authenticators) can be accessed over a transport such
```
Comment (emlun) [inline index.bs]: s/Roaming Authenticators/[=roaming authenticators=]/ ?

File: index.bs (hunk 1 above)
Comment (equalsJeffH) [reply index.bs]: above fine by me if we are caplitalizing those terms consistently

File: index.bs (hunk 2 above)
Comment (equalsJeffH) [reply index.bs]: above fine by me if we are caplitalizing those terms consistently

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs]: Looks like "platform authenticator" is consistently lowercase outside of titles.

File: index.bs (hunk 2 above)
Comment (emlun) [reply index.bs]: Looks like "roaming authenticator" is consistently lowercase.

**Comment by emlun** [comment]
Opened meta-PR selfissued/webauthn#6 with the changes suggested above.

**Pull request merged**